version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.2.0"
```

Comment:
as I was telling earlier, version is automatically managed by sbt-release and should not be changed manually.

Reply:
This is not the case. Repos is manually published to the TrueAccord Artifactory. You, yourself, have bumped the version number in previous commits.

Reply:
they're published to maven central. artifactory is just TA's cache of external maven repos. it appears that I've pumped myself because this is how sbt-release works: when I trigger it from my computer, it pushes separate commit bumping version number: https://github.com/sbt/sbt-release#release-process

Reply:
So you are still publishing with Nadav Sr's credentials. Are you the only one that has them? If so, we need this published. You can set the version number however you want.

Reply:
I'm publishing it with my own credentials, sharing credentials is not a good practice. You need to work with your TA's point of contact to get your own credentials. I've told them some time ago on steps needed to be taken.

Reply:
No one at TA seems to have any knowledge of that, anymore.

Reply:
Who would be the "TA's point of contact"?

Reply:
Person whom you're working with (CTO, TPM, etc) - I don't know about details of your relations with TA.

Reply:
Alright. A different question. Who at TA knows how to create the credentials?

Reply:
I've told this to Andrii and Zhing earlier